network-api/networkapi/nav/models.py

```diff
@@ -50,6 +119,58 @@ class Meta(wagtail_models.TranslatableMixin.Meta):
     def __str__(self) -> str:
         return self.title
 
+    def get_preview_template(self, request, mode_name):
+        return "previews/nav/menu.html"
+
+    def get_preview_context(self, request, mode_name):
+        context = super().get_preview_context(request, mode_name)
+        context["localized_featured_blog_topics"] = self.featured_blog_topics.all().order_by("sort_order")
+        return context
```

Comment:
@danielfmiranda I found this clever way of making preview orderables working again without having to create separate templates. Since context data is checked before properties, the `localized_featured_blog_topics` here takes precedence over the property one on preview requests

Reply:
Wow! Very cool, thanks @jhonatan-lopes